usr.sbin/freebsd-update/freebsd-update.sh

```diff
@@ -1215,6 +1215,18 @@ workdir_init () {
 	touch tINDEX.present
 }
 
+#Fetch compare new metadata version to freebsd-version
+fetch_compare_new_tag_to_freebsd_version () {
+	RUNNINGVERSION=`/bin/freebsd-version`
```

Comment:
This will grab the userland version (i.e. `freebsd-version -u`), if there is a kernel-only update, will this still work?

Reply:
Thank you for your style and code corrections, I've added those.

You question makes me think if there was a kernel only update it would stop it being applied, which is not good.  I don't know how to test for that use case, do you have any suggestions how I can do a mock-up of that scenario?

Reply:
I think this is an easier solution, I wish I thought of this first (seems obvious in hindsight):

https://github.com/freebsd/freebsd-src/pull/543/commits/61c1819ff560a670b1203fec3729371b3e7b17bf

Reply:
I think my latest commit provides a better solution as it just compares tags rather than contents of tags.

Reply:
Nice! Also good that you opened a Bugzilla Bug Report, it should get more visibility than here on GitHub.

Reply:
I think it's looking good now.  Thanks again for your help.